assets/js/main.js

```diff
@@ -1,3 +1,6 @@
+const desktopScreenWidth = 1001;
+const backendURL = 'https://api.codemagic.io'
```

Comment:
This should be environment based. Use Hugo config files to set it via environment variable.